### DISCUSSION TITLE
AEDT/HFSS: Use existing analysis configuration

### DISCUSSION THREAD

**Discussion by SteffenSee92** [ansys/pyaedt#5195] [Q&A]
Hello,

when starting an analysis via analyze (or analyze_setup) method, there is always a new analysis configuration created named pyaedt_config. I tried to change by using the acf_file parameter with the analyze method. The file and settings there seem to be recognized by HFSS (I checked in the GUI, that my new config "myconfig" is set up with proper number of cores as specified in my acf file). But in HFSS, I get then an error message "Invalid Setup "myconfig"" and nothing is solved, but in the script (Jupyterlab) I don't get an error and it says "solved correctly". I also checked in the GUI the settings of the analysis setup "myconfig", and it looks ok (same as default settings). When I click now in the GUI on the analyze button, the setup solves without any errors, using my active analysis configuration "myconfig". It seems, that the use of acf_parameter is buggy at the moment(?).

Is there any other idea how to use an existing analysis configuration instead of generating a new called pyaedt_config by default?

Thanks, best regards

**Comment by Samuelopez-ansys** [comment]
Hi @SteffenSee92 , This bug is fixed in #5199.
